packages/web/src/app/AppProviders.tsx

```diff
@@ -32,27 +32,27 @@ export const AppProviders = ({ children }: AppProvidersProps) => {
   const { history } = useHistoryContext()
   const isMobile = useIsMobile()
 
-  const initialStoreState = {
-    ui: {
-      theme: {
-        theme: getTheme(),
-        systemAppearance: getSystemAppearance()
+  const [{ store, storeHistory }] = useState(() => {
```

Comment:
nice, are we confident that history is initialized correctly when this runs?

Reply:
Local testing showed that it is.
Though [this code](https://github.com/AudiusProject/audius-protocol/blob/0a000c747fed329687b4feb3c04c398c0fd9060d/packages/web/src/app/HistoryProvider.tsx#L31) should probably also be updated to not use `useMemo`, since that isn't guaranteed to never run again after mount.